### DISCUSSION TITLE
Title Inference Coming Soon!

### DISCUSSION THREAD

**Discussion by bramses** [bramses/chatgpt-md#11] [Announcements]
Check it out!

https://user-images.githubusercontent.com/3282661/224617995-7f35c361-0c3b-43ba-9ee3-417c7f50d4ab.mov

**Comment by bramses** [comment]
Live in 1.1.0! 

https://user-images.githubusercontent.com/3282661/224836357-598bf8db-91d0-4c86-a40a-b7e6a9c6034a.mov

**Reply by aaversatile** [reply]
May I ask you what template and font setup are you using in obs? :)

**Reply by bramses** [reply]
Yep!

Template:

```
---
system_commands: ['I am a generalist that is not afraid to make radical associations between distinct topics by using creative leaps.']
temperature: 0.9
top_p: 1
max_tokens: 512
presence_penalty: 1
frequency_penalty: 1
stream: true
stop: null
n: 1
model: gpt-3.5-turbo
---
```

Theme: Typomagical (https://github.com/hungsu/typomagical-obsidian)

**Comment by Nintorac** [comment]
Can't seem to get this to work automatically. I do see title inference in the console after every request though which I imagine is wasting my credits a little too

**Reply by bramses** [reply]
The title inference API call is only called if the regex matches and the list is >4 messages so no worries about tokens (though I will remove that console.log, thanks for the heads up)

Can you try to call it manually with the command "Infer Title" and see what happens?

My guess is for the automatic one the regex is not matching your date format. Can you copy paste the value of that here?

**Reply by Nintorac** [reply]
It is the default `YYYYMMDDhhmmss` I think, and yea calling it manually works
